Automattic/tumblr-theme-garden
# Allow some html in tag_description

### Summary

Tumblr allows some limited set of HTML tags to be rendered in the BlogDescription tag, this PR enables support for that.

### Solution

Setting blogdescription converts it to html entities within Wordpress, so we decode it when handling the tag_description to allow for HTML. We keep wp_kses_post in order to re-filter it to allowed tags.

### Testing steps

1. In site options set tagline to <em>my test desc</em>
2. Save and publish
3. The tumblr theme should render it as italic rather than "<em>my test desc</em>"

### Reviewing tips

🧁 Request a review to team CupcakeLabs to assign a random team member.

🔄 Feel free to ping the team name again to "re-roll" or add another team member.

🚀 For now there is no requirement on getting review approvals before merging.
